dask/tests/test_distributed.py

```diff
@@ -775,3 +775,36 @@ def test_set_index_no_resursion_error(c):
         ddf.compute()
     except RecursionError:
         pytest.fail("dd.set_index triggered a recursion error")
+
+
+@pytest.mark.xfail(reason="https://github.com/dask/dask/issues/8991", strict=True)
```

Comment:
Just an FYI, setting `strict=True` here is totally fine, but we do set it globally in `setup.cfg` 

https://github.com/dask/dask/blob/cac9d6bce45432452d09114daeee5341d1a5305a/setup.cfg#L64